checkov/dockerfile/runner.py

```diff
@@ -9,12 +9,16 @@
 from checkov.dockerfile.registry import registry
 from checkov.runner_filter import RunnerFilter
 
-DOCKER_FILE_MASK = [DOCKERFILE_FILENAME]
+DOCKER_FILE_MASK = r"^(?:.+\.)?[Dd]ockerfile(?:\..+)?$"
 
 
 class Runner(BaseRunner):
     check_type = "dockerfile"
 
+    @staticmethod
+    def __is_docker_file(file):
+        return re.search(DOCKER_FILE_MASK, file) != None
```

Comment:
```suggestion
        return re.match(DOCKER_FILE_MASK, file) is not None
```
a conditional check against `None` should be done with `is`, because this checks the identity and `None` is a singleton.
`match` fits better here, because it checks the string from the beginning against the pattern, compared to `search`, which looks actually somewhere inside the string. There is even `fullmatch`, which would fit this scenario even better 😄

Reply:
Goes to show my inexperience with Python 😉 Thanks a lot for the suggestions, I will push the changes as soon as the tests pass locally!